.circleci/config.yml

```diff
@@ -68,9 +68,10 @@ jobs:
             python setup.py test --installation
             esmvaltool -h
             ncl -V
-            # cdo tests
+            # cdo test, check that it supports hdf5
             cdo --version
-            echo 'netcdf tmp { variables: float test; data: test = 0; }' > tmp.cdl
+            echo 0 | cdo -f nc input,r1x1 tmp.nc
+            ncdump tmp.nc > tmp.cdl
```

Comment:
awesome, dude! That works lovely - but it's a bit of a chicken/egg problem - if we're testing the support of cdo for cdo-created netcdf files , we're not testing support for cdo with generic netcdf files (that currently doesn't work)

Reply:
That's why I use ncdump/ncgen to convert to hdf5 based netcdf4 file